[Doc] Create sensitive-info-notice.md (#1)

grey-olli opened the pull request:
Addedd notes from https://github.com/asbru-cm/asbru-cm/issues/563

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/asbru-cm/asbru-cm?pullRequest=567) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/asbru-cm/asbru-cm?pullRequest=567) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/asbru-cm/asbru-cm?pullRequest=567) it.</sub>

hanspr commented:
I think your warning has a better chance to be read if included in :

`/doc/Manual/Preferences/Preferences.md`

and / or

`/doc/Manual/QuickStart.md`

You may use the expanded `!!! danger "Title"` tag to make it stand up. Check warning on same page at the end.

Or create a link to your new page from any of those pages.

If you are __not__ familiar with mkdocs, check how to install and preview.

https://www.mkdocs.org/#installation

Install, cd to the clone directory on master and execute : `mkdocs serve` 

Preview in http://127.0.0.1:8000/

gfrenoy commented:
Thanks a lot for your help Oleg!  In both cases, it would clearly be useful to have this in our documentation :)

Before merging them, may I indeed ask to:
- Reference to the GitHub guides (https://guides.github.com/activities/forking/)
- If you keep the text, replace "pool request" by "pull request"
- Add your new page to the index (in ```mkdocs.yml```)
- Have a page structure for the new page (main title / section title / ...)

grey-olli commented:
I've fixed typos (thanks @gfrenoy - the 'pool' was very funny one), referenced new page, added minimal text headers. Later if I'll have some time I'll read documentation tool dox noted by @hanspr (thanks!), just now everything made via github web w/o console intervention. Hopefully this makes this pull request better. Mebbe later I'll improve the pull request - currently tight in time.
